rrfs-test/testinput/rrfs_fv3jedi_2024052700_Ens3Dvar.yaml

```diff
@@ -2911,13 +2911,599 @@ cost function:
          #- filter: GOMsaver
          #  filename: ./data/geovals/msonet_geovals_rrfs.nc4
 
+     - obs space:
+         name: atms_npp
+         distribution:
+           name: "RoundRobin"
+           halo size: 100e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: "Data/obs/atms_npp_obs_2024052700_dc.nc"
```

Comment:
I just realized that the each of these files are the large yamls and not templated. Can you update each of these such that the obs space section is a place holder and then the validated yamls get inserted at run time? The atms section alone is almost 600 lines. The entire file length is over 3500 lines. There are 8 of such files. That is a lot of lines we don't need.

The other argument for this is that you don't want to have to maintain these files when someone updates the corresponding "official" yamls.

Edit: Well, I see down below that you might be using the templating. So now I'm confused why we need these yamls since I think they might be generated at run time.

Reply:
@delippi for ctest case, I think we can use the large yamls. All yamls should be treated as fix files so the the developer can run ctest to ensure that there are no error in RDASApp. But we should also include an instruction of how to generate these yamls from JCB(?) so that they may start their development work. We can further discuss this.